salt/metalk8s/saltmaster/minion_configured.sls

```diff
@@ -0,0 +1,31 @@
+{% set saltminion_version = '2018.3.3'%}
+
+
+Configure saltstack repo:
+  pkgrepo.managed:
+    - humanname: SaltStack repo for RHEL/CentOS \$releasever
+    - baseurl: https://repo.saltstack.com/yum/redhat/\$releasever/\$basearch/archive/{{ saltminion_version }}
+    - gpgcheck: 1
+    - gpgkey: https://repo.saltstack.com/yum/redhat/\$releasever/\$basearch/archive/{{ saltminion_version }}/SALTSTACK-GPG-KEY.pub
+
+Install salt minion:
+  pkg.latest:
+    - name: salt-minion
+
+Configure salt minion:
+  file.managed:
+    - name: /etc/salt/minion.d/99-metalk8s.conf
+    - source: salt://metalk8s/saltmaster/files/minion_99-metalk8s.conf.j2
+    - template: jinja
+    - user: root
+    - group: root
+    - mode: '0644'
+    - makedirs: true
+    - backup: false
+
+Restart salt minion:
```

Comment:
Is this 'the right way' @TeddyAndrieux @ballot-scality ?

Reply:
No that doesn't work if you use a master to orchestrate you will basically get no answer look at my PR https://github.com/scality/metalk8s/pull/638/files#diff-023a6ba0457cc7e9ddadf1f92c107fa1